owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/utils/OCFileLoggingTree.kt

```diff
@@ -58,7 +58,11 @@ class OCFileLoggingTree(
             it.list()?.let { logFiles ->
                 if (logFiles.isNotEmpty()) {
 
-                    val lastDateLogFileString = logFiles.last().substringAfterLast("owncloud.").substringBeforeLast(".log")
+                    val lastDateLogFileString = if (context != null) {
+                        logFiles.last().substringAfterLast("${context.packageName}.").substringBeforeLast(".log")
+                    } else {
+                        logFiles.last().substringBeforeLast(".log")
```

Comment:
The name we give a log file in case that `context` is null (shouldn't happen normally) is like `$filename.$fileNameTimestamp.log`. If we consider this, this condition will take `$filename.$fileNameTimestamp`, which is not correct and will lead to a parse exception again